superbit_lensing/pipe.py

```diff
@@ -332,7 +332,8 @@ def run(self, run_options, logprint):
     def _setup_run_command(self, run_options):
 
         galsim_dir = os.path.join(utils.MODULE_DIR, 'galsim')
-        galsim_filepath = os.path.join(galsim_dir, 'mock_superBIT_data.py')
+        #galsim_filepath = os.path.join(galsim_dir, 'mock_superBIT_data.py')
+        galsim_filepath = os.path.join(galsim_dir, 'mock_superBIT_gaussian.py')
```

Comment:
We do not want this checked into the repo

Reply:
Agreed -- I think we could just add an option in the `galsim` config file to have the `optics` convolved with the PSF, `True` by default.